.github/workflows/build.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-20.04, ubuntu-22.04, macos-11, macos-arm-oss ]
+        os: [ ubuntu-20.04, ubuntu-22.04, ubuntu-24.04, macos-11, macos-arm-oss ]
         ruby: [jruby-9.4.7.0]
```

Comment:
Should I add other rubies in this matrix for this case?

Reply:
I typically do it with 1 commit per major.minor, you can see that in existing commits.
IIRC there is some script to easily get such list in this repo or maybe in setup-ruby.
Unfortunately this procedure is not very friendly for PRs.
You could update this one to build all 3.3 releases though, only on 24.04 and if: false on the jruby job below